packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -38,6 +40,22 @@ const testCasesMap = new Map([
 const loggerCategory = "Transformer Performance Regression Tests";
 const outputDir = path.join(__dirname, ".output");
 
+const loadTransformers = async () => {
+  const moduleNames = process.env.EXTRA_TRANSFORMERS?.split(',').map((name) => name.trim()) ?? [];
+  const transformerModules = new Map<string, TestTransformerModule>();
+  transformerModules.set("NativeTransformer", nativeTransformerModule);
+  try {
+    const modules = await Promise.all(
+      moduleNames.map(async moduleName => ({ name: moduleName, module: (await import(moduleName)).default }))
+    );
+    modules.forEach(({ name, module }) => transformerModules.set(name, module));
```

Comment:
I think this would be a bit cleaner without the extra steps and possibly in a more declarative approach:
```ts
// renamed to paths
const modulePaths = process.env.EXTRA_TRANSFORMERS?.split(',').map((name) => name.trim()) ?? [];

const envSpecifiedExtraTransformerCases = await Promise.all(
  modulePaths.map(async (m) => [m, (await import(m).catch(logError)).default])
);

const transformerModules = new Map<string, TestTransformerModule>([
  ["NativeTransformer", nativeTransformerModule],
  ...envSpecifiedExtraTransformerCases
]);
```

Reply:
also do we even need to catch module load error? They'll be rare and user configuration error that isn't super hard to figure out when presented with the crashing callstack.
I'd rather fail the test than keep going if we can't find a specified test transformer